README.md

```diff
@@ -91,23 +117,64 @@ Usage of ./dsv-syncer:
         the Kubernetes namespace containing the Secrets to sync; "" (the default) for all namespaces
 ```
 
-## Build
+### Build
 
-> ___NOTE: Building the `dsv-injector` image is not required to install it as it is___
-> ___available on multiple public registries.___
+> *** note ***
+> Building the `dsv-injector` image is not required to install it as it is.
+> It is available on multiple public registries.
 
-Building the image requires [Docker](https://www.docker.com/)
-or [Podman](https://podman.io/)
-and [GNU Make](https://www.gnu.org/software/make/).
-To build it, run:
+Building the image requires [Docker](https://www.docker.com/) or [Podman](https://podman.io/) and [GNU Make](https://www.gnu.org/software/make/).
 
-```sh
-make
-```
+To build it, run: `make`.
 
 This will build the injector and syncer as platform binaries and store them in the project root.
 It will also build the image (which will build and store its own copy of the binaries) using `$(DOCKER)`.
 
+### Test
+
+The tests expect a few environmental conditions to be met.
+
+> *** note ***
+> For more detailed setup see collapsed section below for DSV Test Configuration Setup.
+
+- A valid DSV tenant.
+- A secret created with the data format below:
+        {
+          "data": {
+            "password": "admin",
+            "username": "admin"
+          },
+          "version": "0"
+        }
+- A `configs/credentials.json` to be created manually that contains the client credentials.
+- The `configs/credentials.json` credential to be structured like this:
+
+    {
+        "app1": {
+            "credentials": {
+                "clientId": "",
+                "clientSecret": ""
+            },
+            "tenant": "app1"
+        }
+    }
+
+> *** warning ***
+> `app1` is required and using any other will fail test conditions.
+
+<details closed>
+<summary>🧪 DSV Test Configuration Setup</summary>
+
+- Using dsv cli (grab from [downloads](https://dsv.secretsvaultcloud.com/downloads) and install with snippet adjusted to version: `$(curl -fSSL https://dsv.secretsvaultcloud.com/downloads/cli/1.35.2/dsv-linux-x64 -o ./dsv && chmod +x dsv-linux-x64 && sudo mv ./dsv-linux-x64 /usr/local/bin/dsv && dsv --version`
+- `dsv init` (Use a local user)
+- Create the role that will allow creating a client for programmatic access: `dsv role create --name 'k8s' --desc 'test profile for k8s'`
+- `dsv secret create --path 'k8s:sync:test' --data '{"password": "admin","username": "admin"}'`
+- Create a policy that allows the local user to read the secret, modify this to the correct user/group mapping: `dsv policy create -- actions 'read' --path 'secrets:k8s' --desc 'test access to secret' --resources 'secrets:k8s:<.*>' --subjects 'roles:k8s'`
```

Comment:
For now I prefer to keep it compact over a block because it's better than what we had, but it's not a fully automated setup at this time, and requires a bit of review. I don't actually want this as a larger block yet.

You can install command runner in vscode by the way and it adds click to run from markdown file for inline blocks like this when you put `run this` in it. 😁